### PR TITLE
Update dough

### PR DESCRIPTION
**Pull request by WalshyDev**
## Description
Updated dough to 1.1.2

## Proposed changes
Updated pom.xml and code in IntegrationsManager

Also updated the changelog, this did fix a few issues with protection things so would be good to be added. However, not sure if it should be in changes or fixes since it's both (though indirectly for fixes)

## Related Issues (if applicable)
N/A

## Checklist
- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Review by svr333 — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -47,6 +47,7 @@
 #### Changes
 * (API) `BiomeMapParser` is now `public`
 * (API) `BiomeMap.fromJson` now allows you to specify if you want the BiomeMap to be parsed leniently
+* Updated dough, fixing some issues
```
Comment (TheBusyBiscuit) [inline CHANGELOG.md:50]: We do not list dependency updates here.
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -47,6 +47,7 @@
 #### Changes
```
Comment (TheBusyBiscuit) [inline CHANGELOG.md:47]: The following needs to be added to the changelog

#### Additions
* Added support for the plugin "HuskTowns"

#### Fixes
* Fixed "LogBlock" integration
* Fixed "Lands" integration

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
You beat me to it. I was just about to update this. But the changelog needs to be fixed before merging this.

File: CHANGELOG.md (hunk 2 above)
Comment (WalshyDev) [reply CHANGELOG.md:47]: Alright, didn't think you'd want them added here since it was in dough but will do

**Review by TheBusyBiscuit — APPROVED** [review]

**Review by svr333 — APPROVED** [review]

**Pull request merged**